fury-on-gke/README.md

```diff
@@ -545,183 +623,134 @@ Navigate to <http://grafana.fury.info> or click the Grafana icon from Forecastle
+# Destroy cluster
+cd /demo/infrastructure
+furyctl cluster destroy
 ```
 
-The output should be something like the following:
-
+4.  Destroy the firewall-rules we have made for nginx ingress:
+  
 ```bash
-$ velero backup get -n kube-system
-NAME   STATUS      ERRORS   WARNINGS   CREATED                         EXPIRES   STORAGE LOCATION   SELECTOR
-test   Completed   0        0          2023-01-02 15:50:16 +0000 UTC   29d       default            <none>
+cd /demo/infrastructure
+make delete-firewall-rule FIREWALL_RULE_NAME=allow-nginx-ingress-admission-webhook
 ```
 
-7. Try to create a pod that is not compliant with the default OPA rules
+5. Destroy network infrastructure (remember to disconnect from the VPN before deleting):
 
 ```bash
-kubectl run --image curlimages/curl curlero -- sleep 500
+cd /demo/infrastructure
+furyctl bootstrap destroy
 ```
 
-The creation should fail with the following error:
+6. (Optional) Destroy the S3 bucket holding the Terraform state
 
 ```bash
-$ kubectl run --image curlimages/curl curlero -n kube-system -- sleep 500
-Error from server (Forbidden): admission webhook "validation.gatekeeper.sh" denied the request: [liveness-probe] Rejecting "Pod/curler" for not specifying a livenessProbe
-[readiness-probe] Rejecting "Pod/curler" for not specifying a readinessProbe
-[enforce-deployment-and-pod-security-controls] container 'curler' in the 'curler' Pod allows priviledge escalation
-[enforce-deployment-and-pod-security-controls] container 'curler' in the 'curler' Pod does not have a CPU limit set
-[enforce-deployment-and-pod-security-controls] container 'curler' in the 'curler' Pod does not have a memory limit set
-[enforce-deployment-and-pod-security-controls] container 'curler' in the 'curler' Pod is running as root
-[enforce-deployment-and-pod-security-controls] container 'curler' in the Pod 'curler' has an image 'curlimages/curl' using the latest tag
+gsutil -m rm -r gs://<GCS_BUCKET>/terraform
+gsutil rb gs://<GCS_BUCKET>
 ```
 
-💡 the `kube-system` namespace is exempted from the OPA rules by default. If you try to create the same Pod in this namespace it should succeed:
+7. Exit from the docker container:
 
 ```bash
-$ kubectl run --image curlimages/curl curler -n kube-system -- sleep 500
-pod/curler created
+exit
 ```
 
-## Step 6 - Teardown
+## Conclusions
 
-To clean up the environment:
+Congratulations, you made it! 🥳🥳
 
-```bash
-# (Required if you performed Disaster Recovery step)
-cd /demo/terraform/demo-fury
-terraform destroy
+We hope you enjoyed this tour of Fury!
 
-# Destroy cluster
-cd /demo/infrastructure
-furyctl cluster destroy
+### Issues/Feedback
 
-# Destroy network components
-# we need to delete the Firewall that we created manually first
-gcloud compute firewall-rules delete allow-nginx-ingress-admission-webhook
-# now we can delete the rest of the resources
-cd /demo/infrastructure
-furyctl bootstrap destroy
+In case your ran into any problems feel free to open an issue here on GitHub.
```

Comment:
```suggestion
In case you ran into any problems feel free to [open an issue in GitHub](https://github.com/sighupio/fury-getting-started/issues/new).
```